### PR TITLE
UIU-2659: When staff user is deleted, then loan history is no longer visible

### PR DESCRIPTION
**Pull request by artem-blazhko**
## Purpose
When user is deleted we are not able to see loan history related to this user.
After merging this PR loan history will be visible.

## Refs
[UIU-2659](https://issues.folio.org/browse/UIU-2659)

## Screenshots

When user and Proxy borrower were deleted

image

When user was deleted  by himself

image

### CODE REVIEW

File: src/views/LoanDetails/LoanDetails.js [hunk 1]
```diff
@@ -320,6 +321,14 @@ class LoanDetails extends React.Component {
     });
   }
 
+  getPageTitle = (loanDetails) => {
+    const { user, patronGroup } = this.props;
```
Comment (Dmitriy-Litvinenko) [inline src/views/LoanDetails/LoanDetails.js:325]:
```
const { 
  user, 
  patronGroup,
 } = this.props;
```

File: src/views/LoanDetails/LoanProxyDetails.js [hunk 2]
```diff
@@ -21,6 +24,21 @@ class LoanProxyDetails extends React.Component {
         records: PropTypes.arrayOf(PropTypes.object),
       }),
     }),
+    mutator: PropTypes.shape({
+      proxy: PropTypes.shape({
```
Comment (Dmitriy-Litvinenko) [inline src/views/LoanDetails/LoanProxyDetails.js:28]: Could you please confirm that proxy should be isRequired?

**Review by Dmitriy-Litvinenko — CHANGES_REQUESTED** [review]

File: src/routes/LoanDetailContainer.js [hunk 3]
```diff
@@ -4,9 +4,16 @@ import PropTypes from 'prop-types';
 import { compose } from 'redux';
 import { concat } from 'lodash';
 import { stripesConnect, withStripes } from '@folio/stripes/core';
+import { Callout } from '@folio/stripes/components';
+import { FormattedMessage } from 'react-intl';
 import { LoanDetails } from '../views';
```
[the remark this answers is not in the document]
Comment (artem-blazhko) [reply src/routes/LoanDetailContainer.js:16]: Fixed

File: src/views/LoanDetails/LoanDetails.js (hunk 1 above)
Comment (artem-blazhko) [reply src/views/LoanDetails/LoanDetails.js]: Fixed

File: src/views/LoanDetails/LoanProxyDetails.js (hunk 2 above)
Comment (artem-blazhko) [reply src/views/LoanDetails/LoanProxyDetails.js:28]: Fixed

**Review by Dmitriy-Litvinenko — APPROVED** [review]

**Review by mkuklis — APPROVED** [review]
Nice work @artem-blazhko!

**Pull request merged**